code/datums/uplink_items/uplink_general.dm

```diff
@@ -213,7 +213,7 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	desc = "A universal gun kit, that can be combined with any weapon kit to make a functioning RND gun of your own. Uses built in allen keys to self assemble, just combine the kits by hitting them together."
 	reference = "IKEA"
 	item = /obj/item/weaponcrafting/gunkit/universal_gun_kit
-	cost = 8
+	cost = 5
```

Comment:
```suggestion
	cost = 25
```
This will now have to be changed in order to conform to the new TC inflation PR that was merged.